designs/developer/add-45/index.md

```diff
@@ -56,7 +56,7 @@ The Pipelines section is available when the Pipelines operator is installed. Des
 ### Advanced Options
 There are a number of Advanced Options which are available, based on the flow:
 - [Routing](https://openshift.github.io/openshift-origin-design/designs/developer/add/sections/Adv-Routing)
-- [Health Checks](https://openshift.github.io/openshift-origin-design/designs/developer/add-45/sections/Adv-Health_Checks)
+- [Health Checks](https://openshift.github.io/openshift-origin-design/designs/developer/add-45/sections/Adv-Health_Checks.)
```

Comment:
@beaumorley should there be a "." after Adv-Health_Checks ?

Reply:
Yes thanks!